scripts/upload_model_to_zenodo.py

```diff
@@ -8,14 +9,13 @@
 from typing import Optional
 from urllib.parse import quote_plus, urljoin, urlparse
 
-import requests  # type: ignore
```

Comment:
I guess you don't get mypy errors about these modules 😅?

Or did you update to typed versions?